ethexe/tx-pool/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "ethexe-tx-pool"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+
+[dependencies]
+anyhow.workspace = true
```

Comment:
reorganise to highlight external and internal deps as its done everywhere else in ethexe

Reply:
done